From 8a8a5de6bbe87c43f97315c31d645df3d795113d Mon Sep 17 00:00:00 2001
From: Jonas Gorski <jonas.gorski@gmail.com>
Date: Sun, 6 Nov 2011 12:33:54 +0100
Subject: [PATCH] MTD: MAPS: bcm963xx-flash.c: explicitly include module.h

module.h was previously implicitly included through mtd/mtd.h.

Fixes the following build failure after the module.h cleanup:

  CC      drivers/mtd/maps/bcm963xx-flash.o
drivers/mtd/maps/bcm963xx-flash.c: In function 'bcm963xx_probe':
drivers/mtd/maps/bcm963xx-flash.c:208:29: error: 'THIS_MODULE' undeclared (first use in this function)
drivers/mtd/maps/bcm963xx-flash.c:208:29: note: each undeclared identifier is reported only once for each function it appears in
drivers/mtd/maps/bcm963xx-flash.c: At top level:
drivers/mtd/maps/bcm963xx-flash.c:255:12: error: 'THIS_MODULE' undeclared here (not in a function)
drivers/mtd/maps/bcm963xx-flash.c:272:16: error: expected declaration specifiers or '...' before string constant
drivers/mtd/maps/bcm963xx-flash.c:272:1: warning: data definition has no type or storage class
drivers/mtd/maps/bcm963xx-flash.c:272:1: warning: type defaults to 'int' in declaration of 'MODULE_LICENSE'
drivers/mtd/maps/bcm963xx-flash.c:272:16: warning: function declaration isn't a prototype
drivers/mtd/maps/bcm963xx-flash.c:273:20: error: expected declaration specifiers or '...' before string constant
drivers/mtd/maps/bcm963xx-flash.c:273:1: warning: data definition has no type or storage class
drivers/mtd/maps/bcm963xx-flash.c:273:1: warning: type defaults to 'int' in declaration of 'MODULE_DESCRIPTION'
drivers/mtd/maps/bcm963xx-flash.c:273:20: warning: function declaration isn't a prototype
drivers/mtd/maps/bcm963xx-flash.c:274:15: error: expected declaration specifiers or '...' before string constant
drivers/mtd/maps/bcm963xx-flash.c:274:1: warning: data definition has no type or storage class
drivers/mtd/maps/bcm963xx-flash.c:274:1: warning: type defaults to 'int' in declaration of 'MODULE_AUTHOR'
drivers/mtd/maps/bcm963xx-flash.c:274:15: warning: function declaration isn't a prototype
drivers/mtd/maps/bcm963xx-flash.c:275:15: error: expected declaration specifiers or '...' before string constant
drivers/mtd/maps/bcm963xx-flash.c:275:1: warning: data definition has no type or storage class
drivers/mtd/maps/bcm963xx-flash.c:275:1: warning: type defaults to 'int' in declaration of 'MODULE_AUTHOR'
drivers/mtd/maps/bcm963xx-flash.c:275:15: warning: function declaration isn't a prototype
drivers/mtd/maps/bcm963xx-flash.c:276:15: error: expected declaration specifiers or '...' before string constant
drivers/mtd/maps/bcm963xx-flash.c:276:1: warning: data definition has no type or storage class
drivers/mtd/maps/bcm963xx-flash.c:276:1: warning: type defaults to 'int' in declaration of 'MODULE_AUTHOR'
drivers/mtd/maps/bcm963xx-flash.c:276:15: warning: function declaration isn't a prototype
make[7]: *** [drivers/mtd/maps/bcm963xx-flash.o] Error 1

Signed-off-by: Jonas Gorski <jonas.gorski@gmail.com>
---

This patch is based on next-20111104 and probably should go through Paul's module.h cleanup tree.

 drivers/mtd/maps/bcm963xx-flash.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

--- a/drivers/mtd/maps/bcm963xx-flash.c
+++ b/drivers/mtd/maps/bcm963xx-flash.c
@@ -21,6 +21,7 @@
 #include <linux/init.h>
 #include <linux/kernel.h>
 #include <linux/slab.h>
+#include <linux/module.h>
 #include <linux/mtd/map.h>
 #include <linux/mtd/mtd.h>
 #include <linux/mtd/partitions.h>
